lib/Sema/DerivedConformanceEquatableHashable.cpp

```diff
@@ -376,34 +1017,64 @@ deriveHashable_enum_hashValue(TypeChecker &tc, Decl *parentDecl,
   //     return index.hashValue
   //   }
   // }
+  //
+  // enum SomeEnumWithAssociatedValues {
+  //   case A, B(Int), C(String, Int)
+  //   @derived var hashValue: Int {
+  //     var result: Int
+  //     switch self {
+  //     case A:
+  //       result = _mixInt(0)
```

Comment:
ps. This change is the best. Sorry I didn't lead with that! :clap::clap::clap::100: